File: app/src/main/java/ru/henridellal/dialer/LogEntryCache.java
package ru.henridellal.dialer;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

public class LogEntryCache {
	public final ImageView callTypeImage;
	public final TextView callDate;
	public final ImageView callSimCard;

	public LogEntryCache(View view) {
		callTypeImage = (ImageView) view.findViewById(R.id.call_type_image);
		callDate = (TextView) view.findViewById(R.id.call_date);
		callSimCard = (ImageView) view.findViewById(R.id.call_sim_card_image);
	}
}

File: app/src/main/java/ru/henridellal/dialer/LogEntryCache.kt
package ru.henridellal.dialer

import android.view.View
import android.widget.ImageView
import android.widget.TextView

class LogEntryCache(view: View) {
    @JvmField
	val contactImage: ImageView = view.findViewById<View>(R.id.contact_image) as ImageView

    @JvmField
	val contactName: TextView = view.findViewById<View>(R.id.contact_name) as TextView

    @JvmField
	val callTypeImage: ImageView = view.findViewById<View>(R.id.call_type_image) as ImageView

    @JvmField
	val phoneNumber: TextView = view.findViewById<View>(R.id.phone_number) as TextView

    @JvmField
	val callDate: TextView = view.findViewById<View>(R.id.call_date) as TextView

}